assets/js/dokan.js

```diff
@@ -2250,6 +2250,10 @@ jQuery(function($) {
           );
 
           $('.dokan-ajax-response').append(resp.data.progress);
+
```

Comment:
remove this code from here, this file will generate automatically via grunt command